### PR TITLE
update nav + lint

### PR DESCRIPTION
**Pull request by just-ma**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  singleQuote: true,
+  printWidth: 120,
+  tabWidth: 2
+};
```
Comment (baolinhkn) [inline .prettierrc.js:7]: do we need a newline here

**Review by baolinhkn — APPROVED** [review]
Thanks for adding the lint :) This is super useful

**Comment by baolinhkn** [comment]
Please add tests next time

File: .prettierrc.js (hunk 1 above)
Comment (just-ma) [reply .prettierrc.js:7]: good catch! it's always great to have another pair of eyes another pair of eyes another pair of eyes eyse another eyes

Comment (baolinhkn) [reply .prettierrc.js:7]: 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀

**Pull request merged**